### ISSUE TITLE
GIF on Twitter breaking Mastodon Crossposting

### ISSUE THREAD

**Issue by ChiefGyk3D** [klausi/mastodon-twitter-sync#74]
I keep getting this issue as of late on a few tweets and can't crosspost anymore using my Raspberry Pi 3B+ that was working reliably. 
I have attempted it with both the rust version and the docker container, same results. Not sure what is happening all of a sudden
```
Error posting toot to Mastodon: Error {
    context: "Failed downloading attachment ",
    source: reqwest::Error {
        kind: Builder,
        source: RelativeUrlWithoutBase,
    },
}
```

**Comment by ChiefGyk3D** [comment]
Ok this seems to be the result of a gif being attached to a tweet, and not being able to brought to Mastodon. It basically kills any future cross posting until I enact

` ./mastodon-twitter-sync --skip-existing-posts`

It works fine with images it seems.

Test Tweets:
Didn't work
https://twitter.com/ChiefGyk3D/status/1631321868136333314

Did work 
https://twitter.com/ChiefGyk3D/status/1631322323285319680

**Comment by klausi** [comment]
What version are you running? I just fixed this issue 2 weeks ago in V1.10.

**Comment by ChiefGyk3D** [comment]
Weird for some reason it wasn't pulling the latest image, it's working now that I forced an update. I will close this issue as it seems to be resolved now
